sechub-webui/src/test/java/com/mercedesbenz/sechub/webui/SecHub/WebUI/SecHubWebUiApplicationTests.java

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui.SecHub.WebUI;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+@EnableConfigurationProperties
+@RunWith(SpringRunner.class)
+@SpringBootTest(value = { "sechub.userid=user", "sechub.apitoken=example" }, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
I think `SpringBootTest.WebEnvironment.RANDOM_PORT`)
is problematic here. 
When starting on jenkins or parallel to our integration tests this would make problems

Reply:
What would be the alternative? Using an environment variable/injecting a value?